src/main/java/io/xpring/ilp/grpc/IlpJwtCallCredentials.java

```diff
@@ -0,0 +1,50 @@
+package io.xpring.ilp.grpc;
+
+import io.grpc.CallCredentials;
+import io.grpc.Metadata;
+import io.grpc.Status;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.Executor;
+
+/**
+ * TODO: Depend on this from hermes?
+ */
+public class IlpJwtCallCredentials extends CallCredentials {
+  private static Logger LOGGER = LoggerFactory.getLogger(IlpJwtCallCredentials.class);
+
+  private final String jwtToken;
+
+  private IlpJwtCallCredentials(String jwtToken) {
+    this.jwtToken = jwtToken;
+  }
+
+  public static IlpJwtCallCredentials build(String token) {
+    return new IlpJwtCallCredentials(token);
+  }
+
+  @Override
+  public void applyRequestMetadata(RequestInfo requestInfo, Executor appExecutor, MetadataApplier applier) {
+    try {
+      applyToken(applier, jwtToken);
+    } catch (RuntimeException e) {
+      applyFailure(applier, e);
+    }
+  }
+
+  @Override
+  public void thisUsesUnstableApi() {}
```

Comment:
What does this mean? Can we add the explanation to the code?

Reply:
`CallCredentials` is an abstract class from gRPC with this method, so we need to either implement it or declare it abstract.